spec/api-web-contents-spec.ts

```diff
@@ -1288,6 +1288,53 @@ describe('webContents module', () => {
     });
   });
 
+  describe('webrtc udp port range policy api', () => {
+    afterEach(closeAllWindows);
+    it('check default webrtc udp port range is {min: 0, max: 0}', () => {
+      const w = new BrowserWindow({ show: false });
+      const settings = w.webContents.getWebRTCUDPPortRange();
+      expect(settings).to.be.an('object');
+      expect(settings).to.have.property('min').that.is.equal(0);
+      expect(settings).to.have.property('max').that.is.equal(0);
+    });
```

Comment:
We can simplify these assertions a bit:

```suggestion
    let w: BrowserWindow;

    beforeEach(() => {
      w = new BrowserWindow({ show: false });
    });

    afterEach(closeAllWindows);

    it('check default webrtc udp port range is { min: 0, max: 0 }', () => {
      const settings = w.webContents.getWebRTCUDPPortRange();
      expect(settings).to.deep.equal({ min: 0, max: 0 });
    });
```

Reply:
Thanks for your guide, which is really helpful for a newbie like me.